CONTRIBUTING.md

```diff
@@ -50,24 +50,32 @@ When you launch the extension or run tests from Visual Studio Code, it will auto
 You can also use these NPM tasks (see `npm run` for the full list):
 
 -   To build once:
+
     ```
     npm run compile
     ```
+
 -   To build and watch for file changes:
+
     ```
     npm run watch
     ```
+
 -   To build a release artifact (VSIX):
+
     ```
     npm run package
     ```
```

Comment:
These extra newlines are a regression IMO. Is there any way to avoid them? It adds gratuitous whitespace:

![image](https://user-images.githubusercontent.com/55561878/124333538-5cbe4100-db49-11eb-9368-abc514fd0066.png)

Reply:
Ah good catch, didn't even notice. Yeah they can be disabled.

Reply:
This was a linter rule not prettier.